test.js

```diff
@@ -19,7 +20,20 @@ describe('Plugin', function() {
     assert.equal(typeof plugin.compile, 'function');
   });
 
+  it('should do nothing for no preset', function (done) {
+    var content = 'var c = {};\nvar { a, b } = c;';
+
+    plugin = new Plugin({ plugins: { babel: { presets: [] }}});
+    plugin.compile({data: content, path: 'file.js'}, function (error, result) {
+      assert(!error);
+      assert(result.data.indexOf(content) !== -1);
+      done();
+    });
+  });
+
   it('should compile and produce valid result', function(done) {
+    this.timeout(10000);
```

Comment:
Don't be sorry, there's no problem. I'm grateful you took the time to update the package.